include/mbgl/shaders/mtl/common.hpp

```diff
@@ -0,0 +1,202 @@
+#pragma once
+
+#define MLN_MTL_UNIFORM_BLOCK(idx, vert, frag, struc) \
```

Comment:
In my opinion it is not worth a saving few extra keystrokes for introducing a macro here.

Reply:
I added this to avoid needing to have the structure name listed twice, once directly and once as a string, and ensure they stay identical.
I was trying to figure out how to make `MLN_STRINGIZE` ensure that the given symbol is valid, but couldn't make it work.

Reply:
How about something like this: https://wandbox.org/permlink/GUq6XSaVVzETU4A4

Reply:
Clever, but that's going to create a lot of changes.  I think it should wait until #1618 is merged, and Stefan's work with string indexing should make some of those obsolete, anyway.

Reply:
OK sure!